ydb/core/protos/flat_scheme_op.proto

```diff
@@ -1000,15 +996,10 @@ message TIndexDescription {
     optional uint64 PathOwnerId = 7;
 
     repeated string DataColumnNames = 8;
+
     // DataSize + IndexSize of indexImplTable
     optional uint64 DataSize = 9;
-
-    // indexImplTable settings
-    oneof Partitions {
-        uint32 UniformPartitions = 10;
-        TExplicitPartitions ExplicitPartitions = 11;
-    }
-    optional TPartitioningPolicy PartitioningPolicy = 12;
+    repeated TTableDescription IndexImplTableDescription = 10;
```

Comment:
According to the [Language Guide (proto 3)](https://protobuf.dev/programming-guides/proto3/) , repeated fields are used in the plural.
```suggestion
    repeated TTableDescription IndexImplTableDescriptions = 10;
```

Reply:
Ok. I agree. More on the issue:
- https://protobuf.dev/programming-guides/style/#repeated-fields
- https://google.aip.dev/144#:~:text=Repeated%20fields%20must%20use%20a,coin%20a%20new%20plural%20form.
- statistically speaking, only 12 out of 94 repeated protobuf field names are singular in flat_scheme_op.proto